airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1362,6 +1378,13 @@ components:
           $ref: "#/components/schemas/JobRead"
         logs:
           $ref: "#/components/schemas/LogRead"
+    DebugRead:
+      type: object
+      required:
+        - info
+      properties:
+        info:
+          type: object
```

Comment:
:) you don't want to make it a structured type?

Reply:
Nope. I want it to be as low friction as possible to add something here and all of the version handling is going to be very temporary as we make changes to how you register integrations.